src/translations/screens/subscreens/AppearanceSettings.ts

```diff
@@ -3,14 +3,15 @@ const AppearanceSettingsTexts = {
   header: {
     title: _('Utseende', 'Appearance', 'Utsjånad'),
   },
-  actions: {
-    usePhoneTheme: _(
-      'Bruk fargetema fra telefonen',
-      "Use my phone's colour theme",
-      'Nytt fargetema frå mobilen min',
-    ),
-    darkMode: _('Mørk modus', 'Dark mode', 'Mørk modus'),
-    useSystemFont: _(
+  theme: {
+    heading: _('Fargetema', 'Color scheme', 'Fargetema'),
```

Comment:
@oyvindmikkelsenatb @Lovise-Git What's a good heading for this setting in English? Is "Color scheme" fine? I've also seen "Theme" and "Color mode" elsewhere.

Reply:
I think I agree, and when there is no response from the designers, I believe the developer on vacation gets to decide! Theme it is 😄🚀

```suggestion
    heading: _('Fargetema', 'Theme', 'Fargetema'),
```